src/front/app/components/ChallengeDetail/ChallengeDetail.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+const AddAndAcceptButton = ({ text, icon }) => (
+    <button className="add-accept-button">
+        <span className="add-accept-button__icon"><img src="" title="logo" />{icon}</span>
+        {text}
+    </button>
+);
+
+const ChallengeCreatedInfo = ({ challengeTitle, createdByTitle, createdBy, rating, challengeLevelTitle, challengeLevel }) => (
+    <section className="challenge-created-info">
+        <div className="challenge-created-info__title">pokormi bomzha {challengeTitle}</div>
+        <div className="challenge-created-info__author-line">
+            <span className="challenge-created-info__author-title">Created by {createdByTitle}</span>
+            <span className="challenge-created-info__author"> Anton Golubev {createdBy}</span>
+            <span className="challenge-created-info__rating"><img src="" title="rating" />{rating}</span>
+        </div>
+        <div className="challenge-created-info__created-by-line">
+            <span className="challenge-created-info__level-title">Challenge Level {challengeLevelTitle}</span>
+            <span className="challenge-created-info__level">Hard {challengeLevel}</span>
+        </div>
+    </section>
+);
+
+const ChallengeCompletedInfo = ({ completedBy, weeks, moreDetails }) => (
```

Comment:
I believe in tasks for HTML markup we can simplify component structure to simple export in component's index file, that would create more convenient way to import markup into components by developers of FE tasks